sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2.scala

```diff
@@ -46,6 +46,9 @@ case class WriteToDataSourceV2(writer: DataSourceWriter, query: LogicalPlan) ext
  * The physical plan for writing data into data source v2.
  */
 case class WriteToDataSourceV2Exec(writer: DataSourceWriter, query: SparkPlan) extends SparkPlan {
```

Comment:
shall we create a new plan for writing micro-batch sink? If we are going to add more streaming related functionalities to this plan, we'd better separate the batch and streaming writing plans.

Reply:
At the moment it is I think overkill. Maybe something to consider for future though as long as we wont find any use for those functionalities we will be adding. The one added now, may would have some use.